modules/odc-adding-health-checks.adoc

```diff
@@ -3,17 +3,17 @@
 // applications/application-health
 
 :_mod-docs-content-type: PROCEDURE
-[id="odc-adding-health-checks"]
+[id="odc-adding-health-checks_{context}"]
 = Adding health checks using the Developer perspective
 
-You can use the *Topology* view to add health checks to your deployed application.
+You can use the *Topology* view to add health checks to your deployed application. 
 
 .Prerequisites:
 * You have switched to the *Developer* perspective in the web console.
 * You have created and deployed an application on {product-title} using the *Developer* perspective.
 
 .Procedure
-. In the *Topology* view, click on the application node to see the side panel. If the container does not have health checks added to ensure the smooth running of your application, a *Health Checks* notification is displayed with a link to add health checks.
+. In the *Topology* view, click the application node to see the side panel. If the container does not have health checks added, a *Health Checks* notification is displayed with a link to add health checks.
 . In the displayed notification, click the *Add Health Checks* link.
 . Alternatively, you can also click the *Actions* drop-down list and select *Add Health Checks*. Note that if the container already has health checks, you will see the *Edit Health Checks* option instead of the add option.
 . In the *Add Health Checks* form, if you have deployed multiple containers, use the *Container* drop-down list to ensure that the appropriate container is selected.
```

Comment:
This isn't part of your PR, but ISG says "Refer to a drop-down list by its label, followed by list." https://www.ibm.com/docs/en/ibm-style?topic=elements-ui#drop_down_list